console/jsre/deps/doc.go

```diff
@@ -18,5 +18,18 @@
 // This file is derived from internal/jsre/deps/deps.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package deps contains the console JavaScript dependencies Go embedded.
+/*
+Package deps contains the console JavaScript dependencies Go embedded.
+
+Klaytn console provides functions of bignumber.js and web3.js which are included in the source tree. To make those available, deps is using go-bindata<https://github.com/jteeuwen/go-bindata> to make those javascript source files into assets which can be used by go environment.
+
+Source Files
+
+Each file provides following features
+  - deps.go	: Define `go:generate` options to use go-bindata in making bignumber.js and web3.js as assets
+  - bindata.go	: Provides functions to load and to get information of assets. This file is generated by go-bindata
+  - bignumber.js	: A JavaScript library for arbitrary-precision decimal and non-decimal arithmetic
```

Comment:
This indentation is ok?

Reply:
@KimKyungup It may seems weird, but it looks okay in web sites. I'm not sure how a browser handle `tab` and `space` in indentation, I changed a bit to fix this indentation and to make it look okay in a browser (Chrome)